dynamic-plugins/wrappers/backstage-community-plugin-redhat-argocd/package.json

```
@@ -0,0 +1,65 @@
+{
+  "name": "backstage-community-plugin-redhat-argocd",
+  "version": "1.6.1",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "private": true,
+  "publishConfig": {
+    "access": "public"
+  },
+  "backstage": {
+    "role": "backend-plugin",
+    "supported-versions": "1.29.2",
+    "pluginId": "backstage-community-plugin-redhat-argocd",
+    "pluginPackages": [
+      "backstage-community-plugin-redhat-argocd"
+    ]
+  },
+  "exports": {
+    ".": "./src/index.ts",
+    "./package.json": "./package.json"
+  },
+  "typesVersions": {
+    "*": {
+      "package.json": [
+        "package.json"
+      ]
+    }
+  },
+  "scripts": {
+    "build": "backstage-cli package build",
+    "lint": "backstage-cli package lint",
+    "test": "backstage-cli package test --passWithNoTests --coverage",
+    "clean": "backstage-cli package clean",
+    "clean-dynamic-sources": "yarn clean && rm -Rf node_modules",
+    "export-dynamic": "janus-cli package export-dynamic-plugin --embed-package @backstage-community/plugin-redhat-argocd --override-interop default --no-embed-as-dependencies"
```

this is a front end plugin. Are you sure you want `--no-embed-as-dependencies` and not --in-place` ? 

For example this other front end pluign: https://github.com/janus-idp/backstage-showcase/blob/main/dynamic-plugins/wrappers/backstage-plugin-azure-devops/package.json#L28

```suggestion
    "export-dynamic": "janus-cli package export-dynamic-plugin --embed-package @backstage-community/plugin-redhat-argocd --override-interop default --in-place"
```

---

FYI

```
$➔ ./node_modules/@janus-idp/cli/bin/janus-cli package export-dynamic-plugin --help
Usage: janus-cli package export-dynamic-plugin [options]

Build and export a plugin package to be loaded as a dynamic plugin. The repackaged dynamic plugin is exported inside a ./dist-dynamic sub-folder.

Options:
  --minify                                                Minify the generated code (backend plugin only).
  --embed-package [package-name...]                       Optional list of packages that should be embedded inside the generated code of a backend dynamic plugin, removed from the plugin
                                                          dependencies, while their direct dependencies will be hoisted to the dynamic plugin dependencies (backend plugin only).
  --shared-package [package-name...]                      Optional list of packages that should be considered shared by all dynamic plugins, and will be moved to peer dependencies of the dynamic
                                                          plugin. The `@backstage` packages are by default considered shared dependencies.
  --override-interop <mode:package-name,package-name...>  Optional list of packages for which the CommonJS Rollup output interop mode should be overridden to `mode` when building the dynamic
                                                          plugin assets (backend plugin only). (default: {})
  --no-install                                            Do not run `yarn install` to fill the dynamic plugin `node_modules` folder (backend plugin only).
  --no-build                                              Do not run `yarn build` on the main and embedded packages before exporting (backend plugin only).
  --clean                                                 Remove the dynamic plugin output before exporting again.
  --dev                                                   Allow testing/debugging a dynamic plugin locally. This creates a link from the dynamic plugin content to the plugin package `src`
                                                          folder, to enable the use of source maps (backend plugin only). This also installs the dynamic plugin content (symlink) into the dynamic
                                                          plugins root folder configured in the app config (or copies the plugin content to the location explicitely provided by the
                                                          `--dynamic-plugins-root` argument).
  --dynamic-plugins-root <dynamic-plugins-root>           Provides the dynamic plugins root folder when the dynamic plugins content should be copied when using the `--dev` argument.
  --embed-as-dependencies                                 Include embedded packages as private dependencies of backend plugins. When value is `false` (using `--no-embed-as-dependencies`), source
                                                          code of embedded plugins is merged with the generated code, so that the embedded plugin packages are completly erased (only available
                                                          for legacy reasons: use with care). (default: true)
  --no-embed-as-dependencies
  --in-place                                              Adds the frontend dynamic plugin assets to the `dist-scalprum` folder of the original plugin package, instead of producing the assets in
                                                          a distinct package located in the `dist-dynamic` sub-folder, as for backend plugins. (default: false)
  --no-in-place
  --scalprum-config <file>                                Allows retrieving scalprum configuration from an external JSON file, instead of using a `scalprum` field of the `package.json`. Frontend
                                                          plugins only.
  -h, --help                                              display help for command
```

---

I guess if another frontend plugin has `--in-place` then so can I, thank you for the suggestion.